otelcollector/configmaps/ama-metrics-settings-configmap.yaml

```diff
@@ -2,7 +2,7 @@ kind: ConfigMap
 apiVersion: v1
 data:
   schema-version:
-    #string.used by agent to parse config. supported versions are {v1}. Configs with other schema versions will be rejected by the agent.
+    #string.used by agent to parse config. supported versions are {v1, v2}. Configs with other schema versions will be rejected by the agent.
```

Comment:
was thinking make this v2, and have a seperate one for v1 and in doing so, keep this file name the same as we have aka.ms links pointing to this one...

Reply:
Makes sense @vishiy .

I've updated this to `ama-metrics-settings-configmap-v1.yaml` and `ama-metrics-settings-configmap.yaml` where the latter is the new v2 configmap